src/views/customer/locales/en.ts

```diff
@@ -0,0 +1,59 @@
+const Customer = {
+  customer: {
+    desEmptyCustomer: 'Manage customer list easily and conveniently',
+    addFiles: 'Add files from excel',
+    addCustomer: 'Add new customer',
+    addCustomers: {
+      title: 'Add new customer',
+      fullName: {
+        label: 'Full name',
+        placeholder: 'Enter first name and last name',
+      },
+      gender: {
+        title: 'Gender',
+        male: 'Male',
+        female: 'Female',
+        other: 'Other',
+      },
+      birthday: 'Date of birth',
+      address: {
+        label: 'Address',
+        placeholder: 'Enter address',
+      },
+      phoneNumber: {
+        label: 'Phone Number',
+        placeholder: 'Enter phone number',
+      },
+      email: {
+        label: 'Email',
+        placeholder: 'Enter email',
+      },
+      employeeSupport: {
+        label: 'Staff in charge',
+        placeholder: 'Staff in charge',
+      },
+      demand: {
+        label: 'Demand',
+        placeholder: 'Demand',
+      },
+      note: {
+        label: 'Note',
+        placeholder: 'Enter note',
+      },
+      btn: {
+        cancel: 'Cancel',
+        submit: 'Add Customer',
+      },
+    },
+    table: {
```

Comment:
It should move to constant for table, we can reuse it

Reply:
But this table is only used in customer and Each table has different fields. i think so